tcl/org.testeditor.tcl.dsl.tests/src/org/testeditor/tcl/dsl/jvmmodel/TclParameterGeneratorIntegrationTest.xtend

```diff
@@ -0,0 +1,95 @@
+package org.testeditor.tcl.dsl.jvmmodel
+
+import javax.inject.Inject
+import org.junit.Before
+import org.junit.Test
+import org.testeditor.aml.AmlModel
+import org.testeditor.aml.dsl.tests.AmlModelGenerator
+import org.testeditor.dsl.common.testing.DummyFixture
+import org.testeditor.tcl.dsl.tests.TclModelGenerator
+import org.testeditor.dsl.common.testing.ResourceSetHelper
+
+class TclParameterGeneratorIntegrationTest extends AbstractTclGeneratorIntegrationTest {
```

Comment:
My opinion: the integration test can be end-to-end / text-to-text. I.e. parse the input, generate, check the output. I would avoid using the model builders here.